app/Services/Scanner.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace App\Services;
+
+use Illuminate\Support\Facades\Process;
+
+/**
+* This service scans configuration values for a Laravel project
+*/
+
+class Scanner
+{
+
+    /**
+     * Scans for templates to generate
+     * 
+     * @return array
+     *      key is template name, and the value is the output file name.
+     */
+    public function templates()
```

Comment:
This isn't meant as a review comment, but just something to be aware of for the future.  The list of templates will likely ultimately be dependent on what options are specified and/or things you determine by scanning the source (and therefore likely are passing to blade).  A prime example would be setting up litefs.  Here's links to corresponding code in the Rails and JS scanners:

* https://github.com/fly-apps/dockerfile-node/blob/a2b2ce4b425ceaf145108bdd5b7d1c3b3e5830f0/gdf.js#L819-L830
* https://github.com/fly-apps/dockerfile-rails/blob/1b9160613654b71a6f9c2d0836d17b8fe407068f/lib/generators/dockerfile_generator.rb#L311-L344
*

Reply:
Gonna add this as a card in the Notes section of the project! Added a [card here](https://github.com/orgs/fly-apps/projects/6/views/1?pane=issue&itemId=56970758), will revise it more later. 

If you want, you can add more notes in the card as well!